app/views/corporate/_content_items.html.erb

```diff
@@ -1,15 +1,9 @@
 <div class="l-inpage-nav">
```

Comment:
I noticed there are two partials doing the same thing, is this correct?
* app/views/corporate/_content_items.html.erb
* app/views/corporate_categories/_content_items.html.erb